firefox-ios/Client/Frontend/Browser/DownloadHelper/Download.swift

```diff
@@ -75,6 +75,12 @@ class Download: NSObject {
         }
         return true
     }
+
+    // Used to avoid name spoofing using Unicode RTL char to change file extension
```

Comment:
Nit: can we have better documentation on this method ? 
```suggestion
    /// Used to avoid name spoofing using Unicode RTL char to change file extension
```